problems/secondSmallest.js

```diff
@@ -6,8 +6,22 @@
 * @returns {number} - Returns the second smallest number.
 */
 
-function secondSmallest() {
-  
+const secondSmallest = arr => {
+    let smallest = Infinity
+    let secondSmallest = Infinity
+    for (let i = 0; i < arr.length; i++) {
+        const element = arr[i];
+        if (element < smallest) {
+            smallest = element
+        }
+    }
+    for (let i = 0; i < arr.length; i++) {
```

Comment:
So you can actually do this is one loop but adding an else if to your if in the first loop that checks if the element is less than secondSmallest